dgf/templates/dgf/includes/friend_ball.html

```diff
@@ -1,6 +1,6 @@
 <a class="friend-ball" href="{% url 'dgf:friend_detail' friend.slug %}">
     {% include "dgf/includes/friend_picture.html" with friend=friend img_size="70px" font_size="30px" %}
     <div class="friend-name">
-        {{friend.short_name}}{% if show_rating and friend.rating %} ({{friend.rating}}){% endif %}
+        {{friend.short_name}}{% if show_rating and friend.rating %} ({{friend.rating}}){% endif %}{% if show_best_score_in_wischlingen and friend.best_score_in_wischlingen %} ({{friend.best_score_in_wischlingen}}){% endif %}
```

Comment:
yo creo que `best_wischlingen_score` dice lo mismo y te ahorras todo un `_in`

Reply:
me parece un cambio to feo pa lo poco que aporta... no se xD

Reply:
quiero decir con esto: me va a costar más cambiarlo de lo que podría dolerme que tuviese `_in`